### PR TITLE
Add pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translate pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

### CODE REVIEW

**Review by Sea-n — COMMENTED** [review]
Please take a look at [Netlify build log](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/62629314d3944900081249f4#L4009), feel free to `/unhold` yourself when the issue is solved.

/hold

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (MrErlison) [reply content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:10]: Thank you. /unhold

**Comment by Sea-n** [comment]
The build error have been solved.
/unhold

It's been a while for this PR, CC pt-approver:
/cc @rikatz

File: content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md:25]: Missing translation for descriptions here.

File: content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md:46]: Very minor: missing `.` at the end of the sentence.

File: content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md:53]: In the "Se kubeadm vazio..." sentence, "empty" refers to the value of the argument. Therefore, the sentence should be "Se vazio, o kubeadm tentará...".

File: content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md:81]: "Um conjunto de locais predefinidos" instead of "um conjunto de padrão local".

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:7]: You used "executa o melhor esforço para reverter" in other places. I think it fits very well here too.

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:12]: This title is talking about the workflow for the reset command in the original documentation. This translation is not really delivering it.

Some suggestions:
* "Fluxo do comando `reset`".
* "Fluxo de execução do comando `reset`".

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:14]: This sentence needs a little bit of rethinking. It's not immediately clear to me from reading this that `kubeadm reset` removes files generated by `init` or `join`.
It's better to use "camada de gerenciamento" instead of "plano de controle".

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:16]: "phase runners" disappeared in the translation, maybe they could be added as "executores de fases dos comandos `kubeadm join`..."?

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:20]: "excluirá nenhum dado **do** etcd" would be more correct.

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:28]: "documentação do etcd".

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:33]: Suggestion: "nó de carga de trabalho".

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @MrErlison, thank you very much for your contribution! 🎉 

I found some items that need to be addressed. Don't hesitate to reach out if you need any clarification.

**Comment by divya-mohan0209** [comment]
Hello @MrErlison , please could you advise if you'd be able to work on this PR and make the suggested changes?

**Comment by MrErlison** [comment]
> Hello @MrErlison , please could you advise if you'd be able to work on this PR and make the suggested changes?

Hi @divya-mohan0209 , I forgot to fix the last suggestion. Sorry about that.

File: content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md:23]: Minor: "verificações pré-execução" would sound better.

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md:18]: Missing translation. Title suggestion: "Limpeza do etcd externo"

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @MrErlison, some very minor things left. Thanks for your contribution!

**Comment by edsoncelio** [comment]
Thanks @stormqueen1990 and @MrErlison 
/approve

**Pull request merged**
